config/deploy.rb

```diff
@@ -18,8 +19,20 @@
 set :rails_env, :production
 set :repo_url, 'https://github.com/18F/identity-dashboard.git'
 set :ssh_options, forward_agent: false, user: 'ubuntu'
+set :bastion_user, 'ubuntu'
 set :tmp_dir, '/tmp'
 
+set :ssh_options do
+  ssh_command = "ssh -A #{fetch(:bastion_user)}@#{fetch(:bastion_host)} -W %h:%p"
```

Comment:
is there a way to override the bastion user on this line? I thought we were supposed to log in to the bastion as ourselves